packages/regenesis-surgery/scripts/handlers.ts

```diff
@@ -101,7 +184,98 @@ export const handlers: {
   [AccountType.UNVERIFIED]: () => {
     return undefined // delete the account
   },
-  [AccountType.VERIFIED]: () => {
+  [AccountType.VERIFIED]: (account, data) => {
```

Comment:
note for @tynes here on possible merge conflicts / operating from the same handler